requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
-boto==2.45.0
+boto==2.34.0
```

Comment:
Oh, sorry, I completely forgot that by the time I got to the end of the code. I don't know of any particular policy either so your approach seems sensible.